### ISSUE TITLE
Violation of 4.1.1, if with ARIA syntax is correct?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2135]
The following code is syntactically incorrect (according to the W3C validator), thus a violation of SC 4.1.1, but not incorrect per se due to the ARIA markup. 

```
<ul>
  <div role=listitem></div>
</ul>
```

`listitem` is equivalent to `<li>`. Should this still be considered a violation of 4.1.1 - and if so: why?

**Comment by mraccess77** [comment]
I'm not sure I would consider it a 4.1.1 violation - but I'd like to others opinions as well.

**Comment by cstrobbe** [comment]
When WCAG 2.0 was in its final year of development, development of WAI-ARIA had already started, with a [first public draft of WAI-ARIA published in February 2008](https://www.w3.org/TR/2008/WD-wai-aria-20080204/). At the time, it was impossible to use WAI-ARIA attributes in HTML (still HTML 4.01 at the time) or XHTML without causing validation errors. (This changed later when HTML 5 drafts began to explicitly address the inclusion of WAI-ARIA attributes in HTML documents.)

It was never WCAG's intention to prohibit the use of WAI-ARIA attributes by means of the wording of SC 4.1.1. For this reason, there is nothing in the SC that prohibits custom attributes.

The code snippet from the original issue does not contain anything that violates SC 4.1.1, neither at the level of element nesting (but that's a separate discussion), nor at the level of attribute syntax:

    <ul>
      <div role=listitem></div>
    </ul>

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do that and convert it to a Discussion.
